templates/parachain/README.md

```diff
@@ -47,90 +70,143 @@ git clone https://github.com/paritytech/polkadot-sdk-parachain-template.git para
 cd parachain-template
 ```
 
-### Build
+## Starting a Development Chain
+
+### Omni Node
 
-🔨 Use the following command to build the node without launching it:
+[Omni Node](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/reference_docs/omni_node/index.html) can
+be used to run the parachain template's runtime. `polkadot-omni-node` binary crate usage is described at a high-level
+[on crates.io](https://crates.io/crates/polkadot-omni-node).
+
+#### Install `polkadot-omni-node`
+
+Please see the installation section at [`crates.io/omni-node`](https://crates.io/crates/polkadot-omni-node).
+
+#### Build `parachain-template-runtime`
 
 ```sh
-cargo build --release
+cargo build -p parachain-template-runtime --release
 ```
 
-🐳 Alternatively, build the docker image:
+#### Install `staging-chain-spec-builder`
+
+Please see the installation section at [`crates.io/staging-chain-spec-builder`](https://crates.io/crates/staging-chain-spec-builder).
+
+#### Use `chain-spec-builder` to generate the `chain_spec.json` file
 
 ```sh
-docker build . -t polkadot-sdk-parachain-template
+chain-spec-builder create --relay-chain "rococo-local" --para-id 1000 --runtime \
+    <target/release/wbuild/path/to/minimal-template-runtime.wasm> named-preset development
 ```
 
-### Local Development Chain
+**Note**: the `relay-chain` and `para-id` flags are mandatory information required by
+Omni Node, and for parachain template case the value for `para-id` must be set to `1000`, since this
+is also the value injected through [ParachainInfo](https://docs.rs/staging-parachain-info/0.17.0/staging_parachain_info/)
+pallet into the `parachain-template-runtime`'s storage.
 
-🧟 This project uses [Zombienet](https://github.com/paritytech/zombienet) to orchestrate the relaychain and parachain nodes.
-You can grab a [released binary](https://github.com/paritytech/zombienet/releases/latest) or use an [npm version](https://www.npmjs.com/package/@zombienet/cli).
+#### Run Omni Node
 
-This template produces a parachain node.
-You can install it in your environment by running:
+Start Omni Node with the generated chain spec.
 
 ```sh
-cargo install --path node
+polkadot-omni-node --chain <path/to/chain_spec.json> 
```

Comment:
The OmniNode setup here should not demonstrate a broken command, instead it should: 

1. explain that a parachain cannot progress without relay chain, which we can only setup with ZN
2. use `--dev-block-time` here.